src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.test.tsx

```diff
@@ -513,6 +515,72 @@ it("toggles between the product offerings in the premium upsell dialog", async (
   expect(productMontlyCta).toBeInTheDocument();
 });
 
+it("counts in Glean how ofen people click the upgrade CTA to purchase the yearly plan", async () => {
+  const user = userEvent.setup();
+  const ComposedDashboard = composeStory(
+    DashboardUsNoPremiumNoScanNoBreaches,
+    Meta,
+  );
+  const mockedRecord = useTelemetry();
+  render(<ComposedDashboard />);
+
+  // We show a CTA on desktop in the toolbar and in the mobile menu
+  const premiumCtas = screen.queryAllByRole("button", {
+    name: "Upgrade to ⁨Premium⁩",
+  });
+  await user.click(premiumCtas[0]);
+  const productTabMonthly = screen.getByRole("tab", { name: "Monthly" });
+  await user.click(productTabMonthly);
+  const confirmButton = screen.getByRole("link", {
+    name: "Select monthly plan",
+  });
+  // jsdom will complain about not being able to navigate to a different page
+  // after clicking the link; suppress that error, as it's not relevant to the
+  // test:
+  jest.spyOn(console, "error").mockImplementationOnce(() => undefined);
+  await user.click(confirmButton);
+
+  expect(mockedRecord).toHaveBeenCalledWith(
+    "ctaButton",
+    "click",
+    expect.objectContaining({
+      button_id: "intent_to_purchase_monthly_plan_nav_modal",
+    }),
+  );
+});
+
+it("counts in Glean how ofen people click the upgrade CTA to purchase the monthly plan", async () => {
```

Comment:
```suggestion
it("counts in Glean how often people click the upgrade CTA to purchase the monthly plan", async () => {
```

Reply:
Same Q as above re: test name being "purchase the monthly plan" and the test selecting the yearly plan.

Reply:
237105aca